civihr_default_theme/civihr_default_theme.info

```diff
@@ -29,6 +29,7 @@ scripts[] = assets/js/civihr_default_theme.script.js
 scripts[] = assets/js/jquery.matchHeight-min.js
 scripts[] = assets/js/jquery.customSelect.min.js
 
+
```

Comment:
initally added js file there but then removed it, are you asking about additional line?